### PR TITLE
Changed the global vars to $GLOBALS and fix some includes

### PR DESCRIPTION
**Pull request by byjg**
I had to use the project as a composer module and ran into some problems. 

The first problem was the use of "global" directive. Inside my project, the system does no recognized these variables. So I changed to use $GLOBALS.

The second problem I had been in the include of files. Because the use of relative paths I got some problems. I added to the path the directive `__DIR__` to force the use of the correct directory.

### CODE REVIEW

**Comment by pentium10** [comment]
I am curious what PHP installation you ran when you hit this problems. It should complain all of these. Maybe your php.ini is heavy customized or secured safe.

On other notes, we try to keep this version of beanstalk_console compatible with PHP5.2, and __DIR__  is not yet supported by 5.2 

There are plans to feature froze the project and to rewrite into more modern PSR-4 standard with namespace support, but lack of time is delaying it.

**Comment by byjg** [comment]
Thank you for answer me. I did the follow:

```
cd /to/myproject
composer require "ptrofimov/beanstalk_console=dev-master"
ln -s $PWD/vendor/ptrofimov/beanstalk_console/public/ httpdocs/admin
```

The folder "httpdocs" is my public web root. 

I have also a friendly url names and a route.php file for process all requests. 

The route.php file do this:

```php
 if (!empty($_SERVER['SCRIPT_FILENAME']) && file_exists($_SERVER['SCRIPT_FILENAME'])) {
    $file = $_SERVER['SCRIPT_FILENAME'];
    if (strpos($file, '.php') !== false) {
        require_once($file);
    } else { 
        /* Do other stuff */ 
    }
} 
```

And I call `http://myhost/admin/index.php`

In this environment, I had problems. 

Do you have any other suggestion for I put your project inside mine as a composer package?

**Comment by byjg** [comment]
If you accept my changes I can replace `__DIR__` to `dirname(__FILE__)`. 

Also, I can help you to change your project to a PSR-4 compliant.

**Comment by byjg** [comment]
Hey, I did the changes you asked. Any updates about this pull request?

**Comment by pentium10** [comment]
I tried to do it, but the project fails to start after these commits and didn't had the time to check why.

Maybe is because you added `require_once 'Pheanstalk/ClassLoader.php';` and there is no such file.

**Comment by byjg** [comment]
Hi, I double checked my commit and I not found runtime errors neither as a package or a single server. 

```
git clone git@github.com:byjg/beanstalk_console
git checkout changeglobal
composer update
php -S localhost:8080
```

Everything works. 

Note: I am relying on composer.

File: lib/include.php [hunk 1]
```diff
@@ -10,17 +10,20 @@ function __autoload($class) {
 }
 
 session_start();
+require_once 'Pheanstalk/ClassLoader.php';
```
Comment (pentium10) [inline lib/include.php]: @byjg why is this referenced here, we don't have this file in the repo.

Comment (byjg) [inline lib/include.php]: Yes, you have. 

See:
https://github.com/ptrofimov/beanstalk_console/blob/master/lib/Pheanstalk/ClassLoader.php

**Pull request merged**